### ISSUE TITLE
Custom macros

### ISSUE THREAD

**Issue by Justinicus** [kiibohd/KiiConf#76]
Will the Configurator ever support custom macros the way QMK/QMK Firmware Builder do?

**Comment by DoWhileGeek** [comment]
Yes, I would very much like the configurator to support custom macros, I really dont want to compile myself just for macros.
